### PR TITLE
Transifex

### PR DESCRIPTION
**Pull request by mkllnk**
#### What? Why?

Merging translations in every release cycle.

#### What should we test?

Nothing.

#### Release notes

Not necessary.

#### How is this related to the Spree upgrade?

Not at all.

### CODE REVIEW

**Comment by daniellemoorhead** [comment]
@mkllnk who do you think is best to code review this?

**Comment by mkllnk** [comment]
@daniellemoorhead No code review needed. We just merge it without testing. It has just not been automated yet, so I have to do it manually.

**Comment by sigmundpetersen** [comment]
@mkllnk @daniellemoorhead should we have a nice label for this? maybe `skip code review`? does this imply "not in release notes" or should we have a separate label for that as well?

**Comment by mkllnk** [comment]
I added that label. But I'm not sure if it implies "not in the release notes". For example, we could improve the documentation like the README file and include that in the release notes. That could be reviewed though.

**Comment by daniellemoorhead** [comment]
Sally usually adds a list at the bottom of release notes of all the non-user centric stuff that goes live in the release, and I'd categorise this as one of them. Worth knowing it's in the release, not worth knowing much more than that.

File: config/locales/es.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@ es:
   activerecord:
     attributes:
       spree/order:
-        payment_state: Estado del pago
+        payment_state: Estado del pagos
```
Comment (enricostano) [inline config/locales/es.yml]: cc/ @sseerrggii should we fix this in Transifex?

Comment (sseerrggii) [inline config/locales/es.yml]: :+1:

**Comment by daniellemoorhead** [comment]
@mkllnk will this be part of the release today or should it be moved to September?

**Pull request merged**